exercises/bank-account/src/main/scala/Bank.scala

```diff
@@ -0,0 +1,13 @@
+trait BankAccount {
+
+  def closeAccount(): Unit
+
+  def getBalance: Option[Int]
+
+  def incrementBalance(increment: Int): Option[Int]
+}
+
+object Bank {
```

Comment:
I changed the name for the Object from BankAccount to Bank. I didn't like having a BankAccount.openAccount function. It seems like a Bank should be opening accounts not a BankAccount.  I thought about having a Bank.scala and BankAccount.scala. But, then I didn't like having 2 .scala files for the exercise..